Tests/Model/CardPackage/DeckTest.java

```diff
@@ -23,16 +23,16 @@
      * This resets the decks before every test
      ***********************************************************************************/
     @Before
-    void setUp(){
-        chanceDeck = new Deck(System.getenv("CHANCE_FILE"),true);
-        communityDeck = new Deck(System.getenv("COMMUNITY_FILE"),false);
+    public void setUp(){
+        chanceDeck = new Deck("res/chance.txt",true);
+        communityDeck = new Deck("res/community.txt",false);
     }
 
     /**********************************************************************************
      * Test the functionality of the drawCard method
      ***********************************************************************************/
     @Test
-    void drawCard() {
+    public void drawCard() {
```

Comment:
Since we where just doing testing I made every method package private. Would wy would we need them to be public ?

Reply:
I was getting error related to the method names not having a scope when I went to run the test. I had to define the scope in order for the test to run.

Reply:
If I don't have them public I get the following error. 

`java.lang.Exception: Method [method name] should be public`